elm/config/elm_cfg.yaml

```diff
@@ -1,20 +1,22 @@
 model: codegen-350M-mono
 checkpoints_dir: ./checkpoints/
-device: cuda
+cuda: True
```

Comment:
When I run, it still asks for `device` key. Also it asks for `eval_steps`, `n_bins` and more.